src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -29,45 +30,88 @@ public void setEncryptor(Encryptor encryptor) {
         this.encryptor = encryptor;
     }
 
-    public boolean accountValidation(Account account){
-        if(account.getName().isEmpty() || account.getPassword().isEmpty() || account.getEmail().isEmpty() || account.getUsername().isEmpty() || account.getAddress().isEmpty() || account.getPhoneNumber().isEmpty() || account.getDateOfBirth().isEmpty())
-            return false;
+    public ValidationResponse accountValidation(Account account){
+
+        ValidationResponse validationResponse = new ValidationResponse();
+
+        if(StringUtils.isEmpty(account.getName()))
```

Comment:
maybe we should group validations for a specific field and extract them to separate methods to increase readability. Also, do we need different messages for example in the phone number is empty or contains invalid characters? Can't we go with a single and more generic one?